tests/e2e/lib.sh

```diff
@@ -594,7 +594,7 @@ wait_for_collectors_to_be_operational() {
                 echo "$pod is deemed ready"
             else
                 info "$pod is not ready"
-                kubectl -n "${sensor_namespace}" logs -c collector "$pod"
+                kubectl -n "${sensor_namespace}" logs -c collector "$pod" | true
```

Comment:
I think, we want to use `||` here instead of piping the result to file name `true` 😉 
```suggestion
                kubectl -n "${sensor_namespace}" logs -c collector "$pod" || true
```

Reply:
🤦